### ISSUE TITLE
[Feature] generate other editor configs using pnpify --sdk

### ISSUE THREAD

**Issue by jeysal** [yarnpkg/berry#1231]
- [ ] I'd be willing to implement this feature
- [ ] This feature can already be implemented through a plugin

**Describe the user story**
Setting up PnP support for coc.nvim as documented on https://yarnpkg.com/advanced/editor-sdks currently requires manually creating a config file after the `yarn pnpify --sdk` step.

**Describe the solution you'd like**

The `sdk` option could take an argument telling it what to generate the sdk for, e.g. `--sdk={vscode,coc.nvim,...}`.

**Describe the drawbacks of your solution**

Potential feature creep by supporting every editor in existence.

**Describe alternatives you've considered**

Plain `--sdk` could just generate them all. Would unfortunately create lots of unneeded files in the repo. Although current `--sdk` already generates `.vscode` even for non VSCode users :shrug: 

**Additional context**

From https://github.com/yarnpkg/berry/issues/170#issuecomment-617129234

**Comment by zypA13510** [comment]
**Describe the user story**

I would like to add support of pnpified eslint to Atom. However, it feels quite strange for an Atom plugin to rely on the existence of `.vscode` to function properly.

**Describe the solution you'd like**

Maybe move the default `.vscode/pnpify` to a neutral location, like `.yarn/pnpify`, and then use `--sdk` flag to generate config required for each SDK.

**Describe the drawbacks of your solution**

- it would be hard to clean up existing installations
- any existing SDK that uses fixed location `.vscode/pnpify` instead of a config file will be broken

**Comment by esetnik** [comment]
I didn’t find this when I created #1441 so sorry for the duplicate. I’m willing to help out with this. I’m a coc.nvim user and found some friction when trying out yarn berry for the first time. I personally feel like the sdk should be changed to be editor agnostic. What that means for me is that by default it should just generate a .yarn/pnpify directory and place the detected pnpified libs in there without adding any additional editor config. I don’t think vscode by default is good as it sets a precedent that this tool is somehow promoting use of a specific editor. I also think there’s a valid use case for generating all editor configs with a single command. Consider, for example an open source project that wants to make it easy for contributors no matter which IDE they are using. So I proposed separate switches for each config which could be combined to generate any combination of editor configs in a single shot.
